IMOS_ATF-ACOUSTIC/index_tmp.sql

```diff
@@ -0,0 +1,47 @@
+-- insert what has been harvested into the database with index_tmp
+-- creates if needed the table detection_file_index
+-- then creates the views to be used for WMS
+
+CREATE TABLE IF NOT EXISTS detection_file_index
+(
+    -- id bigserial PRIMARY KEY,
+    tag_deployment_id TEXT PRIMARY KEY, --UNIQUE NOT NULL,
+    transmitter_id TEXT,
+    species TEXT,
+    tag_project TEXT,
+    time_coverage_start timestamp with time zone,
+    time_coverage_end timestamp with time zone,
+    geom geometry(Geometry,4326),
+    first_indexed timestamp with time zone DEFAULT Now(),
+    last_indexed timestamp with time zone DEFAULT Now(),
+    url TEXT,
+    size integer
+);
+
+-- -- create the geometry column for a polygon
+-- --SET search_path = harvest, public;
+-- SELECT AddGeometryColumn('animal_tracking_acoustic_qc','index_tmp','geom',4326,'POLYGON',2);
+-- UPDATE animal_tracking_acoustic_qc.index_tmp
+--        SET geom = ST_GeomFromText('POLYGON(('|| min_lon::text ||' '|| min_lat::text ||','|| min_lon::text ||' '|| max_lat::text ||',
+-- '|| max_lon::text ||' '|| max_lat::text ||','|| max_lon::text ||' '|| min_lat::text ||', '|| min_lon::text ||' '|| min_lat::text ||'))', 4326);
```

Comment:
Not essential but at some point you could clean up these comments if they're no longer needed?